tests/kernel/threads/dynamic_thread/prj.conf

```diff
@@ -1,3 +1,3 @@
 CONFIG_ZTEST=y
 CONFIG_TEST_USERSPACE=y
-CONFIG_HEAP_MEM_POOL_SIZE=8192
+CONFIG_HEAP_MEM_POOL_SIZE=20000
```

Comment:
Must it be this big?  I guess all our userspace platforms are mostly unconstrained, but there are still small systems in the world that might want to run this test.